### DISCUSSION TITLE
erreur sur l'accès au site

### DISCUSSION THREAD

**Discussion by Lb59** [dbarzin/mercator#103] [General]
Bonjour,
Nous avons redemarré le serveur ubuntu et je ne suis plus autorise à accede au site.
Meme sur le serveur en localhost j'ai perdu le site.
Le fichier mercator.conf est correct, l'affectation du site est faite, et le serveur apache est bien démarré ?
que puis je faire ?

merci

**Comment by dbarzin** [comment]
Cela peut être lié à la verson de php
Aller dans le répertoire de Mercator et faire un "composer update"

**Comment by dbarzin** [comment]
Vérifier les logs qui se trouvent dans storage/logs/laravel.log

**Comment by dbarzin** [comment]
Vérifier les logs apache
Vérifier qu'Apache écoute sur le bon port
Vérifier les sites Apache actifs
